uber/templates/dept_checklist/form.html

```diff
@@ -8,8 +8,8 @@
 {{ conf.description }}
 <br/> <br/>
 <input type="hidden" name="slug" value="{{ item.slug }}" />
-{% csrf_token %}
-<textarea name="comments" rows="5" cols="80">{{ item.comments|default:"" }}</textarea>
+{{ csrf_token() }}
```

Comment:
This doesn't need to be done for this PR, but I wonder if the csrf_token can be tied into the 'admin area' flag somehow.